packages/nimble-components/src/button/tests/button.react.tsx

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I'd vote to add configuration to eslintrc rather than inline disables in each file. You should be able to do that with an `overrides` rule targeting `*.react.tsx`. That would also allow you to add a comment justifying the exceptions.